token/cli/src/main.rs

```diff
@@ -356,13 +359,22 @@ async fn command_create_token(
     }
 
     let freeze_authority = if enable_freeze { Some(authority) } else { None };
+    let mut extensions = vec![];
+    if let Some(rate_bps) = rate_bps {
+        extensions.push(
+            spl_token_client::token::ExtensionInitializationParams::InterestBearingConfig {
+                rate_authority,
+                rate: rate_bps,
+            },
+        )
+    }
```

Comment:
youll have to rebase against master because we create this vector now (as part of the close-authority pr), but you shouldnt have to do anything different otherwise